changes.go

```diff
@@ -251,10 +251,7 @@ type CommentInput struct {
 }
 
 // DiffIntralineInfo entity contains information about intraline edits in a file.
-type DiffIntralineInfo []struct {
-	SkipLength int
-	MarkLength int
-}
+type DiffIntralineInfo [][2]int
```

Comment:
Previously, the pair of numbers were named as `SkipLength` and `MarkLength`. Now that it's just an array of two ints, it's harder to know what they mean. Perhaps it's a good idea to compensate by adding documentation (from https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#diff-intraline-info)? E.g.:

```Go
// DiffIntralineInfo entity contains information about intraline edits in a file.
//
// The information consists of a list of <skip length, mark length> pairs,
// where the skip length is the number of characters between the end of
// the previous edit and the start of this edit, and the mark length is the
// number of edited characters following the skip. The start of the edits
// is from the beginning of the related diff content lines.
//
// Note that the implied newline character at the end of each line
// is included in the length calculation, and thus it is possible for
// the edits to span newlines.
type DiffIntralineInfo [][2]int
```